### PR TITLE
cart displays 1 ton of selected material

### LINKED ISSUE #29: Space cart should listen for radio button clicks on Facility Minerals
When the user clicks a Facility Mineral radio button, should add 1 ton of <mineral> to the space cart

May be the same or a different transient state

### PR DESCRIPTION
**Pull request by thegoodcorg**
close #29

### CODE REVIEW

**Review by bigjlr2004 — APPROVED** [review]
This smells like awesome!!!!!

**Review by loganallengray — APPROVED** [review]
👍

File: scripts/SpaceCart.js [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+const minerals = getMinerals()
+const facilityInv = getFacilityInventories()
+
+document.addEventListener("click", (e) => {
+    if (e.target.name === "facilityInventory") {
+        const spaceCart = {}
+        for (const inv of facilityInv) {
+            if (inv.id === parseInt(e.target.value)) {
+                spaceCart.amount = 1
```
Comment (will-hedges) [inline scripts/SpaceCart.js:11]: If this stays at 1, I don't think we necessarily need to define it as a variable

File: scripts/SpaceCart.js [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+                for (const mineral of minerals) {
+                    if (mineral.id === inv.mineralId) {
+                        spaceCart.item = mineral.type
+                    }
+                }
+            }
+        }
+        const spaceCartElem = document.querySelector("#space-cart__container");
+        spaceCartElem.innerHTML = `${spaceCart.item}: ${spaceCart.amount}`;
```
Comment (will-hedges) [inline scripts/SpaceCart.js:20]: needs to say "ton" at the end

**Review by will-hedges — CHANGES_REQUESTED** [review]

**Pull request merged**